src/nxdoc/nuxeo-server/data-store/persistence-architecture/file-storage.md

```diff
@@ -408,20 +408,26 @@ This assumes that you have four blob providers configured, the default one and t
 ```xml
 <extension target="org.nuxeo.ecm.core.blob.BlobManager" point="configuration">
   <blobprovider name="videos">
-    <class>org.nuxeo.ecm.core.blob.binary.DefaultBinaryManager</class>
+    <class>org.nuxeo.ecm.core.blob.LocalBlobProvider</class>
     <property name="path">binaries-videos</property>
   </blobprovider>
   <blobprovider name="attachments">
-    <class>org.nuxeo.ecm.core.blob.binary.DefaultBinaryManager</class>
+    <class>org.nuxeo.ecm.core.blob.LocalBlobProvider</class>
     <property name="path">binaries-attachments</property>
   </blobprovider>
   <blobprovider name="encrypted">
-    <class>org.nuxeo.ecm.core.blob.binary.AESBinaryManager</class>
+    <class>org.nuxeo.ecm.core.blob.AESBlobProvider</class>
     <property name="key">password=secret</property>
   </blobprovider>
 </extension>
 ```
 
+{{#> callout type='warning' heading='Separating Binaries'}}
+It is CRITICAL to keep the binaries separated between each provider if you want orphaned binaries to be properly deleted. If you don't do this, things will seem to work correctly but orphaned binaries GC may incorrectly delete data.
+
+Always define different `path` when using local blob providers. When using [Amazon S3 Online Storage](https://connect.nuxeo.com/nuxeo/site/marketplace/package/amazon-s3-online-storage) (or any other cloud provider), always define different `bucket_prefix` ( or container prefix) if using the same bucket (or container).
```

Comment:
To avoid rephrasing differently maybe:
"Always define different `path` when using local blob providers.
When using cloud storage (such Amazon S3) always different locations, see <nuxeo-retention-installation-standard.md critical section> "

Reply:
I prefer to keep stating this `WARN` as is on this page and not point to the AWS s3 retention config page.

This consideration is valid for any storage solution, including Azure, GCP, etc.. (hence the term container)